website/docs/d/nsxt_distributed_firewall_rule.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_distributed_firewall_rule"
+sidebar_current: "docs-vcd-data-source-nsxt-distributed-firewall-rule"
+description: |-
+  The Distributed Firewall data source a single rules for a particular VDC Group.
+---
+
+# vcd\_nsxt\_distributed\_firewall\_rule
+
+The Distributed Firewall data source reads all defined rules for a particular VDC Group.
```

Comment:
I'm a bit confused here with the word "all". I thought this is for reading a single rule, as the `name` ir required?

Reply:
Fixed.

Reply:
There are some issues with the latest sentence :)

> The Distributed Firewall data source a single rules for a particular VDC Group.

->

> The Distributed Firewall data source **reads** a single **rule of** a particular VDC Group.

Reply:
Thank you. That wasn't good - reread both docs now :)